### ISSUE TITLE
[Graphics] OpenGoal v0.1.9 - prologue flickering graphics

### ISSUE THREAD

**Issue by skullpanda86** [open-goal/jak-project#1521]
### Describe the Bug

https://youtu.be/JPK14SSUIdg

I updated OpenGoal to v0.1.9, went to run to check some stuff and noticed the prologue was flickering, tried updating my nvidia drivers and it was still doing the same thing.

### How To Reproduce

Running it.

### Expected Behavior

Too not flicker during the prologue.

### Operating System

Windows 11

### OpenGOAL Version

Release

### Game Version

NTSC 1.0 (black label)

**Comment by ManDude** [comment]
By flickering do you mean the particles on the dark eco mist were constantly shifting between in front of or behind the mist?

**Comment by BreakPoints** [comment]
Yeah, the particles are jumping from in front and behind a 4:3 black box that's used for fading the cutscene in and out of black, maybe even darken the scene. Blending looks borked.

![image](https://user-images.githubusercontent.com/11966090/175170361-ce55f200-32bf-413e-8cad-5669423a1094.png)

**Comment by water111** [comment]
there's not supposed to be a black box there at all, no idea what that's about

**Comment by water111** [comment]
any chance that https://github.com/open-goal/jak-project/blob/master/goal_src/levels/village1/sequence-a-village1.gc#L89 requires some change to the sparticle asm stuff that's only in pal?

**Comment by water111** [comment]
are you on a > 60 Hz refresh rate monitor?

**Comment by BreakPoints** [comment]
> are you on a > 60 Hz refresh rate monitor?

I am, yes. Using the NTSC build also so it shouldn't be a PAL issue.

**Comment by skullpanda86** [comment]
> are you on a > 60 Hz refresh rate monitor?

So I did a little further testing today and yeah I think the issue is tied to the users monitor refresh rate, because as soon I tried enabling vsync and it stopped, the only probably is as soon you reopen the game you have to manually reenable vsync.

**Comment by BreakPoints** [comment]
That makes sense as the Intel device that didn't have the issue was already limited to 60Hz

**Comment by BreakPoints** [comment]
Flickering is fixed at 150FPS as of 0.1.12 but the black box remains. I think it just has to be scaled for wider aspect ratios

**Comment by ManDude** [comment]
Yeah, that would do it. I'm leaving all of the stupid garbage sprite scaling and positioning for last because it's annoying to deal with.
